pkg/storage/storage.go

```diff
@@ -314,6 +317,9 @@ type ChangelogBackend interface {
 	// if no changes are found, it should return storage.ErrNotFound and an empty continuation token.
 	// It the objectType and the type in the continuation token don't match, it should return ErrMismatchObjectType.
 	ReadChanges(ctx context.Context, store string, filter ReadChangesFilter, options ReadChangesOptions) ([]*openfgav1.TupleChange, []byte, error)
+
+	// SerializeReadChangesContToken serializes the continuation token into a format readable by ReadChanges
+	SerializeReadChangesContToken(ulid string, objType string) ([]byte, error)
```

Comment:
The addition of `SerializeReadChangesContToken` into this interface seems kinda leaky imho. It's usage outside of storage adapter implementations themselves is in `pkg/server/commands/read_changes.go`.

I wonder if there is an alternative implementation where we don't need this interface change?